agent/bench-scripts/pbench-fio

```diff
@@ -157,15 +157,15 @@ function fio_usage() {
 		printf -- "\t--remote-only\n"
 		printf "\t\trun this on the remotes only\n"
 		printf "\n"
-		printf -- "\t--histogram-interval-msec=<int>\n"
-		printf "\t\tset the histogram logging interval in milliseconds (default $histogram_interval_msec)\n"
+		printf -- "\t--histogram-interval-sec=<int>\n"
```

Comment:
Can we keep the old parameter option for compatibility?

Reply:
fio-histo-log-pctiles.py doesn't support sub-second time quantum, Andrew seemed to think it was ok not to go below 1 second.  I don't think a lot of people were using histogram logging up to now.  

I could keep the old parameter name and just convert it into seconds, and not document it (deprecate it).  Is that good enough.

Reply:
Yes, keeping the old parameter name and just convert it into seconds, and not documenting it (deprecating it), seems fine.